src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -1031,6 +1031,18 @@
     <value>The url of the hosted icon file</value>
   </data>
   <data name="MultipleMatchingNestedInstallersFound_Error" xml:space="preserve">
-    <value>Multiple matches found for "{0}" from {1}</value>
+    <value>Multiple matches found for "{0}" from {1}. Please use the interactive mode to update this package.</value>

```

Comment:
Need an explanation for `{0}` and `{1}` similar to what is done on line 968. Same for the ones below.